modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -255,43 +237,194 @@ public function testCreateListPages() {
     $search_button->click();
-    $this->assertSearchResultsTitle('Event list page', 4);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 4);
+    $this->assertResults([
       'Event number 2',
       'Event number 4',
       'Event number 9',
       'Event number 11',
     ]);
+
+    // Test Project list page.
+    $date_plus_1 = date('Y-m-d', strtotime('+1 day'));
+    $date_plus_10 = date('Y-m-d', strtotime('+10 days'));
+
+    Node::create([
+      'title' => 'Project closed',
+      'type' => 'oe_project',
+      'oe_summary' => 'This is a closed Project',
+      'language' => 'en',
+      'status' => NodeInterface::PUBLISHED,
+      'oe_project_budget' => 100,
+      'oe_project_dates' => [
+        'value' => '2020-05-10',
+        'end_value' => '2020-05-15',
+      ],
+      'oe_subject' => 'http://data.europa.eu/uxp/1000',
+    ])->save();
+
+    Node::create([
+      'title' => 'Project ongoing',
+      'type' => 'oe_project',
+      'oe_summary' => 'This is a ongoing Project',
+      'language' => 'en',
+      'status' => NodeInterface::PUBLISHED,
+      'oe_project_budget' => 33,
+      'oe_project_dates' => [
+        'value' => '2022-05-20',
+        'end_value' => $date_plus_1,
+      ],
+      'oe_subject' => 'http://data.europa.eu/uxp/1567',
+    ])->save();
+
+    Node::create([
+      'title' => 'Project pending',
+      'type' => 'oe_project',
+      'oe_summary' => 'This is a pending Project',
+      'language' => 'en',
+      'status' => NodeInterface::PUBLISHED,
+      'oe_project_budget' => 1234,
+      'oe_project_dates' => [
+        'value' => $date_plus_1,
+        'end_value' => $date_plus_10,
+      ],
+      'oe_subject' => 'http://data.europa.eu/uxp/1018',
+    ])->save();
+
+    // Index content.
+    $this->indexItems('oe_list_pages_index');
+
+    $list_page = $this->createListPage('oe_project', [
+      'oelp_oe_sc_project__budget' => 'oelp_oe_sc_project__budget',
+      'oelp_oe_sc_project__end_date' => 'oelp_oe_sc_project__end_date',
+      'oelp_oe_sc_project__start_date' => 'oelp_oe_sc_project__start_date',
+      'oelp_oe_sc_project__status' => 'oelp_oe_sc_project__status',
+      'oelp_oe_sc_project__type' => 'oelp_oe_sc_project__type',
+    ]);
+
+    $this->drupalGet($list_page->toUrl());
+
+    $this->assertResultsTitle('Results', 3);
+    $this->assertResults([
+      'Project closed',
+      'Project ongoing',
+      'Project pending',
+    ]);
+
+    // Assert that the filter form for Events exists.
+    $filter_form = $assert_session->elementExists('css', '#oe-list-pages-facets-form');
+    $filter_status = $filter_form->findField('Status');
+    $filter_type = $filter_form->findField('Project type');
+    $filter_budget = $filter_form->findField('Total budget');
+    $filter_start = $filter_form->findField('Start date');
+    $filter_end = $filter_form->findField('End date');
+    $search_button = $filter_form->find('css', '#edit-submit');
```

Comment:
We forgot one assertNotNull():
```suggestion
    $search_button = $filter_form->find('css', '#edit-submit');
    $this->assertNotNull($filter_status);
```
My IDE complains if a variable could be NULL..